website/addons/github/serializer.py

```diff
@@ -0,0 +1,48 @@
+from website.addons.base.serializer import OAuthAddonSerializer
+from website.util import api_url_for, web_url_for
+
+
+class GitHubSerializer(OAuthAddonSerializer):
+
+    @property
+    def serialized_urls(self):
+        external_account = self.node_settings.external_account
+        ret = {
+            'auth': api_url_for('oauth_connect',
+                                service_name=self.node_settings.provider_name),
```

Comment:
I'm not sure, this is the same pattern I've seen in the other external provider classes. I'll talk to lyndsy about edge cases.